Mlem/App/Utility/Extensions/Content Models/Comment1Providing+Extensions.swift

```diff
@@ -188,4 +192,34 @@ extension Comment1Providing {
             callback: callback
         )
     }
+    
+    func collapseAction(commentTreeTracker: CommentTreeTracker? = nil) -> BasicAction {
+        .init(
+            id: "collapse\(uid)",
+            appearance: .collapse(),
+            callback: { @MainActor in
+                commentTreeTracker?.nodesKeyedByActorId[self.actorId]?.collapsed = true
+            }
+        )
+    }
+    
+    func collapseParentAction(commentTreeTracker: CommentTreeTracker? = nil) -> BasicAction {
+        .init(
+            id: "collapseParent\(uid)",
+            appearance: .collapseParent(),
+            callback: { @MainActor in
+                commentTreeTracker?.nodesKeyedByActorId[self.actorId]?.parent?.collapsed = true
```

Comment:
This should be `.toggle()` - the user should be able to swipe to un-collapse when the comment is collapsed.
```suggestion
                commentTreeTracker?.nodesKeyedByActorId[self.actorId]?.parent?.collapsed.toggle()
```

(Ditto for the other collapse actions)